src/main/kotlin/blackJack/domain/Result.kt

```diff
@@ -1,23 +1,22 @@
 package blackJack.domain
 
-class Result(private val dealer: Dealer, players: Players) {
-    private val _dealerResult = WinOrLose.values().map { it to DEFAULT }.toMap().toMutableMap()
-    val dealerResult
-        get() = _dealerResult.toMap()
-    val playersResult = players.players.map { it to getWinOrLose(it) }.toMap()
+class Result(players: Players, private val dealer: Dealer) {
+    private var _dealerProfit: Int = DEFAULT_PROFIT
+    val dealerProfit
+        get() = _dealerProfit
```

Comment:
프로퍼티의 getter 함수를 노출하면서 setter 함수에 대한 접근을 제한하려면 setter 함수를 private으로 지정하면 됩니다.

```suggestion
    var dealerProfit: Int = DEFAULT_PROFIT
        private set
```